actions/entrypoint.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/sh -l
```

Comment:
@qianz2 do we need to add this file? Any changes that we're making to our GitHub Action should be done in its repository, [`appservice-build`](https://github.com/Azure/appservice-build)

Reply:
Gotcha, I'm not sure where to commit my changes at first. It not necessary for oryx to have this file, I'll remove it later :)